service/controller/machinepool/handler/migration/testdata/case-1-New-MachinePool-status-not-updated_MachinePool.golden

```diff
@@ -0,0 +1,91 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
New MachinePool CR that got created and the migration is not complete (CR references are updated). New MachinePool migration handler then updates Status, so here we see that the new MachinePool CR is fully created and the migration is done.